index.html

```diff
@@ -236,6 +314,8 @@ <h3>Skills</h3>
                         <button class="c-button" type="submit">Get in touch</button>
                     </form>
```

Comment:
Also, for your contact section, kindly fix it so it is as close to the design as possible 👍 

Your design :
![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/20537712/2dca49ae-6f8a-4b1b-b859-cf9711b7a168)

Figma design :

![image](https://github.com/allenkays/Portfolio-setup-and-mobile-first/assets/20537712/14ad20e5-fa10-4279-ae88-4628c70cdc00)